ppx_cstruct.opam

```diff
@@ -23,5 +23,6 @@ depends: [
   "ocaml-migrate-parsetree"
   "ppx_driver"    {test & >= "v0.9.0"}
   "ppx_sexp_conv" {test}
+  "cstruct-unix"  {test}
```

Comment:
Yeah, I'm not sure why the container-based ones do not run it by default. I explicitly add it when TESTS are desired (some repositories have non-terminating tests, as older tcpip revisions)